addons/website_slides/models/slide_slide_resource.py

```diff
@@ -19,6 +20,7 @@ class SlideResource(models.Model):
     file_name = fields.Char(store=True)
     link = fields.Char('Link', compute='_compute_reset_resources', store=True, readonly=False)
     download_url = fields.Char('Download URL', compute='_compute_download_url')
+    sequence = fields.Integer(default=10)
```

Comment:
Can we specify the label manually?

Also, would not specify the default to 10.
I know people tend to specify 10 but I suspect it to be a copy/pasted mistake, there is really no point.